# HG changeset patch
# User Rik <rik@octave.org>
# Date 1500860080 25200
#      Sun Jul 23 18:34:40 2017 -0700
# Node ID 99cdc1547054094a0474d19541b03c867aeaa099
# Parent  f864950f837c942b164b55adcc0895e558f76279
Stop unused parameter warning when building without FLTK.

* __init_fltk__.cc (F__init_fltk__): Use octave_unused_parameter() in #else
branch when compiling without FLTK.

diff --git a/libinterp/dldfcn/__init_fltk__.cc b/libinterp/dldfcn/__init_fltk__.cc
--- a/libinterp/dldfcn/__init_fltk__.cc
+++ b/libinterp/dldfcn/__init_fltk__.cc
@@ -2507,16 +2507,18 @@ Undocumented internal function.
       octave_value_list id = Fadd_input_event_hook (fcn_handle, 1);
 
       fltk->set_input_event_hook_id (id);
     }
 
   return octave_value_list ();
 
 #else
+  octave_unused_parameter (interp);
+
   err_disabled_feature ("__init_fltk__", "OpenGL and FLTK");
 #endif
 }
 
 /*
 ## No test needed for internal helper function.
 %!assert (1)
 */
